Test needs updating:

```
FAIL: For contracted invoices, emails should be sent to finance@dimagi.com
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/accounting/tests/test_invoicing.py", line 258, in test_contracted_invoice_email_recipient
    self.assertEqual(actual_recipient, expected_recipient)
AssertionError: Lists differ: ['finance@dimagi.com', 'accoun... != ['finance@dimagi.com']
First list contains 1 additional elements.
First extra element 1:
'accounts@dimagi.com'
- ['finance@dimagi.com', 'accounts@dimagi.com']
+ ['finance@dimagi.com']
```

--- a/corehq/apps/accounting/tests/test_invoicing.py
+++ b/corehq/apps/accounting/tests/test_invoicing.py
@@ -249,7 +249,7 @@ class TestContractedInvoices(BaseInvoiceTestCase):
         For contracted invoices, emails should be sent to finance@dimagi.com
         """
 
-        expected_recipient = ["finance@dimagi.com"]
+        expected_recipient = ["finance@dimagi.com", "accounts@dimagi.com"]
 
         tasks.generate_invoices(self.invoice_date)
 

